CraftingRevisions/TestBlueprints.cs

```diff
@@ -0,0 +1,59 @@
+﻿
+// just some json blueprints for testing 
+
+namespace CraftingRevisions
+{
+	internal class TestBlueprints
+	{
+
+		public static string blueprint_1 = """
+						{
+			    "Name": "Old_Prybar",
+			    "RequiredGear": ["GEAR_Cloth"],
+			    "RequiredGearUnits": [1],
+			    "KeroseneLitersRequired": 0,
+			    "GunpowderKGRequired": 0,
+			    "RequiredTool": "",
+			    "OptionalTools": [],
+			    "RequiredCraftingLocation": "Anywhere",
+			    "RequiresLitFire": false,
+			    "RequiresLight": false,
+			    "CraftedResult": "GEAR_Prybar",
+			    "CraftedResultCount": 1,
+			    "DurationMinutes": 15,
+			    "CraftingAudio": "PLAY_CraftingCloth",
+			    "AppliedSkill" : "None",
+			    "ImprovedSkill" : "None"
+			}
+			""";
+
+		public static string blueprint_2 = """
+			{
+			"Name":"UserBp_Prybar",
```

Comment:
Is name part of the Hinterland blueprints, or something you added for use in validation validation, similar to the old format?